qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/views/CreateCustomerView.groovy

```diff
@@ -158,18 +164,31 @@ class CreateCustomerView extends FormLayout {
         })
 
         this.submitButton.addClickListener({ event ->
-            String firstName = this.firstNameField.getValue().trim()
-            String lastName = this.lastNameField.getValue().trim()
-            String title = this.titleField.getValue()?.trim() ?: "None"
-            String email = this.emailField.getValue().trim()
-            List<Affiliation> affiliations = new ArrayList()
-            if (affiliationComboBox.selectedItem.isPresent() ) {
-                affiliations.add(affiliationComboBox.getSelectedItem().get())
-            }
-            if (allValuesValid()) {
-                controller.createNewCustomer(firstName, lastName, title, email, affiliations)
-            } else {
-                this.viewModel.failureNotifications.add("Please fill out the customer information correctly.")
+            try {
+                String title
+                String firstName = this.firstNameField.getValue().trim()
+                String lastName = this.lastNameField.getValue().trim()
+                String email = this.emailField.getValue().trim()
+                List<Affiliation> affiliations = new ArrayList()
+                if (affiliationComboBox.selectedItem.isPresent()) {
+                    affiliations.add(affiliationComboBox.getSelectedItem().get())
+                }
+                if (titleField.selectedItem.isPresent()) {
+                    title = titleField.selectedItem.get()
+                } else {
+                    title = ""
```

Comment:
NIT: Why don't we set the title to "None" here, instead of the controller?
```suggestion
                    title = "None"
```

Reply:
I would want to avoid intrinsic knowledge of the enum from being in the view and rather move it to the controller. This way we do not get invalid values after changing the view in the future